src/DependencyInjection/WebpackEncoreExtension.php

```diff
@@ -51,6 +51,9 @@ public function load(array $configs, ContainerBuilder $container)
         $container->getDefinition('webpack_encore.entrypoint_lookup_collection')
             ->replaceArgument(0, ServiceLocatorTagPass::register($container, $factories));
         $container->setAlias(EntrypointLookupInterface::class, new Alias($this->getEntrypointServiceId('_default')));
+
+        $container->getDefinition('webpack_encore.tag_renderer')
+            ->replaceArgument(2, $config['crossorigin']??'anonymous');
```

Comment:
The default value should be moved to the Configuration class. This way, tooling can now about it.

Reply:
Done.